### ISSUE TITLE
Classification code

### ISSUE THREAD

**Issue by King4819** [uzh-rpg/svit#3]
Hi, I want to ask that whether the code of classification parts is completed yet. Seems like the code now is only for testing, but I want to implement full training (30 epochs in your paper) to reproduce the result,. Thanks a lot and this is such an awesome work !!!

**Comment by kaikai23** [comment]
Hi, here is the original command I used to produce the result:
<pre><code>main.py --model exp1p_v1_gumbel_R7_small --fuse_token --base_keep_rate 0.7 --input-size 224 --sched cosine --lr 1e-5 --min-lr 1e-6 --weight-decay 1e-6 --batch-size 256 --shrink_start_epoch 0 --warmup-epochs 0 --shrink_epochs 0 --epochs 30 --dist-eval --finetune /net/cephfs/shares/rpg.ifi.uzh/yifei/output/deit_small_patch16_224-cd65a155.pth --data-path ../datasets/ImageNet1k/ --output_dir /net/cephfs/shares/rpg.ifi.uzh/yifei/output/train --exp_name selective-s-finetune30-4x256-lr1e-5-minlr-1e-6 --ratio-loss-lambda 4 --disable-pre-multiply-mask --post-multiply-mask --select-cls-for-last-layer --change-key-name-from-timm-to-pytorch</code></pre>
The code was cleaned a bit before release, so some args need to be chaned, e.g. `--model svit_s`

**Comment by King4819** [comment]
@kaikai23 I think the main problem in classification code is that it doesn't seperate training and inference stage like svit det code. This is why I am curious about whether the code is completed verison.
